tests/integration/test_zones_setpoint.py

```
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)